xbmc/windowing/X11/GLContextEGL.h

```diff
@@ -35,6 +35,11 @@ class CGLContextEGL : public CGLContext
   void SetVSync(bool enable) override;
   void SwapBuffers() override;
   void QueryExtensions() override;
+  void CheckBufferAgeSupport() override
```

Comment:
maybe make this return a bool instead and adapt the main logic?

Reply:
Done that, but I think it is a bit confusing now.